scripts/verifyCommit.js

```diff
@@ -0,0 +1,26 @@
+// Invoked on the commit-msg git hook by yorkie.
+
+const chalk = require('chalk');
+const msgPath = process.env.GIT_PARAMS;
```

Comment:
`msgPath` is undefined on my MacOS, please take a look.

Reply:
sure @juzhiyuan I will check this.